src/main/java/com/exasol/adapter/dynamodb/mapping/reader/ColumnMappingReader.java

```diff
@@ -10,16 +10,23 @@
  */
 class ColumnMappingReader {
     private static final ColumnMappingReader INSTANCE = new ColumnMappingReader();
-    private static final String MAX_LENGTH_KEY = "maxLength";
-    private static final int DEFAULT_MAX_LENGTH = 254;
-    private static final String OVERFLOW_KEY = "overflow";
-    private static final String OVERFLOW_ABORT = "ABORT";
-    private static final String DEST_NAME_KEY = "destName";
+    private static final String VARCHAR_COLUMN_SIZE_KEY = "varcharColumnSize";
+    private static final int DEFAULT_VARCHAR_COLUMN_SIZE = 254;
+    private static final String OVERFLOW_BEHAVIOUR_KEY = "overflowBehaviour";
+    private static final String ABORT_KEY = "ABORT";
+    private static final String NULL_KEY = "NULL";
+    private static final String DEST_NAME_KEY = "destinationName";
     private static final String REQUIRED_KEY = "required";
-    private static final String TO_STRING_MAPPING_KEY = "toStringMapping";
+    private static final String TO_VARCHAR_MAPPING_KEY = "toVarcharMapping";
     private static final String TO_JSON_MAPPING_KEY = "toJsonMapping";
+    private static final String TO_DECIMAL_MAPPING_KEY = "toDecimalMapping";
     private static final TruncateableMappingErrorBehaviour DEFAULT_TO_STRING_OVERFLOW = TruncateableMappingErrorBehaviour.TRUNCATE;
     private static final MappingErrorBehaviour DEFAULT_LOOKUP_BEHAVIOUR = MappingErrorBehaviour.NULL;
+    private static final String DECIMAL_PRECISION_KEY = "decimalPrecision";
+    private static final String DECIMAL_SCALE_KEY = "decimalScale";
+    private static final int DEFUALT_DECIMAL_SCALE = 0;
+    private static final int DEFAULT_DECIMAL_PRECISION = 18;
+    private static final String NOT_NUMERIC_BEHAVIOUR = "notNumericBehaviour";
```

Comment:
Maybe you need a separate class for constants. That's quite a lot here

Reply:
Well but they are not used outside of the class. Should I still move them? I think it would somehow seperate things that belong together?

Reply:
You could add an inner non-instantiable class that only stores constants and move it down. I just don't like that the constants are taking so much space in this class and they distract me from what the class is actually doing 
But if you feel okay about how it looks now you can leave it be. It's just a taste question probably